python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -19,12 +19,15 @@
 import numpy as np
 import tvm
 
+import tvm._ffi
```

Comment:
Better to use more specific import
```suggestion
from tvm._ffi import register_func
```

Reply:
Done